errors/MinimumInvestment.js

```diff
@@ -0,0 +1,8 @@
+const ApplicationError = require('./ApplicationError');
+
+class MinimumInvestment extends ApplicationError {
+  constructor(message) {
+    super(message || 'El minimo monto a invertir es de 3500.', 400);
```

Comment:
Should this be depending on risk instead of a constant 3500?

Reply:
Oh I get it, its just a placeholder. If you see, it does `message || El minimo monto a invertir es de 3500.` 

So the proper value gets introduced later on in the code. But it is indeed confusing

Reply:
Oh I get it, its just a placeholder. If you see, it does `message || El minimo monto a invertir es de 3500.` 

So the proper value gets introduced later on in the code. But it is indeed confusing

Reply:
Yes you are right!